.circleci/config.yml

```diff
@@ -976,13 +982,53 @@ workflows:
             branches:
               only:
                 - irs
-                - migration
                 - staging
                 - master
                 - experimental1
                 - experimental2
                 - experimental3
                 - dawson
+  build-and-deploy-without-smoketests:
+    jobs:
+      - build-shared:
+          <<: *build-and-deploy-without-smoketests-defaults
+      - build-api:
+          <<: *build-and-deploy-without-smoketests-defaults
+      - build-client-unit:
+          <<: *build-and-deploy-without-smoketests-defaults
+      - build-client-integration:
+          <<: *build-and-deploy-without-smoketests-defaults
+      - e2e-pa11y:
+          <<: *build-and-deploy-without-smoketests-defaults
+      - e2e-pa11y-public:
+          <<: *build-and-deploy-without-smoketests-defaults
+      - e2e-cypress:
+          <<: *build-and-deploy-without-smoketests-defaults
+      - e2e-cypress-public:
+          <<: *build-and-deploy-without-smoketests-defaults
+      - build-client-coverage:
+          <<: *build-and-deploy-without-smoketests-defaults
+          requires:
+            - build-shared
+            - build-api
+            - build-client-unit
+            - build-client-integration
+            - e2e-pa11y
+            - e2e-pa11y-public
+            - e2e-cypress
+            - e2e-cypress-public
+      - deploy:
+          <<: *build-and-deploy-without-smoketests-defaults
+          requires:
+            - build-client-coverage
+      - migrate:
+          <<: *build-and-deploy-without-smoketests-defaults
+          requires:
+            - deploy
+      - switch-colors:
```

Comment:
No need to change for the PR, but you might wish to change this workflow so it doesn't switch colors if there is a migration. Might take a bit of logic. 

For deployments that require a migration, that migration can take about an hour to complete. The "wait for migration to finish" step did not timeout and fail, and so the workflow continued to switch colors prematurely.